klyshko-operator/config/samples/klyshko_v1alpha1_tuplegenerationscheduler.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: klyshko.carbnyestack.io/v1alpha1
+kind: TupleGenerationScheduler
+metadata:
+  name: tuplegenerationscheduler-sample
+spec:
+  concurrency: 2
```

Comment:
There is nothing concrete planned as of now. Something that comes to my mind would be a scheduler that takes the tuple consumption rate into consideration as well. That would allow to create tuples of those types first that are predicted to become scarce first based on available tuples and consumption rate. Yes, I guess the CRD would evolve over time but that's okay as the API is marked alpha so far.